taskfiles/lint/yaml.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+
+tasks:
+  check-yaml:
+    desc: "Runs the YAML linters."
+    sources:
+      - "{{.G_LINT_VENV_CHECKSUM_FILE}}"
+      - "{{.ROOT_DIR}}/**/*.yaml"
+      - "{{.ROOT_DIR}}/**/*.yml"
+      - "{{.TASKFILE}}"
```

Comment:
```suggestion
```

* Like in the GH workflow, let's not include `.yml` files.
* Tracking changes to `.yaml` files should already cover `{{.TASKFILE}}`.